ts/components/IdpsGrid.tsx

```diff
@@ -116,6 +116,7 @@ const renderItem =
 
 const IdpsGrid: React.FunctionComponent<Props> = (props: Props) => (
   <FlatList
+    scrollEnabled={false}
```

Comment:
Why don't we pass this through props?

Reply:
@dmnplb

Reply:
Okay, if this is the only place where we use this component, we could leave it as it is. If in the future we need to enable/disable the scroll from the parent component, we'll pass it through the `props`.